src/main/resources/lang/dutch.lang

```diff
@@ -0,0 +1,190 @@
+		all:
+			disabled: Alle scripts zijn met succes uitgeschakeld.
+			io error: <light red>Een of meer scripts konden niet hernoemd worden - sommige scripts zijn mogelijk al hernoemd en worden uitgeschakeld wanneer de server opnieuw wordt opgestart: <gold>%s
+		single:
+			already disabled: <gold>%s<reset> is al uitgeschakeld!
+			disabled: <gold>%s<reset> is succesvol uitgeschakeld.
+			io error: <light red><gold>%s<light red> kon niet worden hernoemd. Het wordt weer ingeschakeld wanneer je de server opnieuw opstart: <gold>%s
+		folder:
+			empty: <gold>%s<reset> bevat geen ingeschakelde scripts.
+			disabled: <gold>%2$s<reset> script(s) in <gold>%1$s<reset> zijn met succes uitgeschakeld.
+			io error: <light red>Een of meerdere scripts in <gold>%s konden <light red>niet worden uitgeschakeld <light red>(sommige scripts worden mogelijk uitgeschakeld wanneer de server opnieuw opstart): <gold>%s
+	update:
+		# check/download: see Updater
+		changes:
+			# multiple versions:
+			# 	title: <gold>%s<r> update¦ is¦s zijn¦ uitgebracht sinds deze versie (<gold>%s<r>) van Skript:
+			# 	footer: Om de changelog van een versie te tonen gebruik je <gold>/skript update changes <version><reset>
+			# invalid version: Geen changelog voor de versie <gold>%s<red> beschikbaar
+			title: <bold><cyan>%s<reset> (%s)
+			next page: <grey>pagina %s van %s. Gebruik <gold>/skript update changes %s<gray> voor de volgende pagina (hint: gebruik de pijltjestoets omhoog)
+	info:
+		aliases: Skript's aliassen kunnen hier gevonden worden: <aqua>https://github.com/SkriptLang/skript-aliases
+		documentation: Skript's documentatie kan hier gevonden worden: <aqua>https://docs.skriptlang.org/
+		tutorials: Skript's handleidingen kunnen hier gevonden worden: <aqua>https://docs.skriptlang.org/tutorials
+		version: Skript Versie: <aqua>%s
+		server: Server Versie: <aqua>%s
+		addons: Geïnstalleerde Skript addons: <aqua>%s
+		dependencies: Geïnstalleerde afhankelijkheden: <aqua>%s
+
+# -- Updater --
+updater:
+	not started: Skript heeft nog niet gecontroleerd of je de nieuwste versie hebt. Gebruik <gold>/skript update check<reset> om dit te doen.
+	checking: Zoeken naar de nieuwste versie van Skript...
+	check in progress: Er wordt momenteel gecontroleerd op een nieuwe versie.
+	updater disabled: Het updateprogramma is uitgeschakeld, dus er is niet gecontroleerd op de nieuwste versie van Skript.
+	check error: <red>Er is een fout opgetreden bij het controleren op de nieuwste versie van Skript:<light red> %s
+	running latest version: Je gebruikt momenteel de nieuwste stabiele versie van Skript.
+	running latest version (beta): Je gebruikt momenteel een <i>bèta<r> versie van Skript en er is geen nieuwe <i>stabiele<r> versie beschikbaar. Houd er rekening mee dat je handmatig moet updaten naar nieuwere bètaversies!
+	update available: Er is een nieuwe versie van Skript beschikbaar: <gold>%s<reset> (je gebruikt momenteel <gold>%s<reset>)
+	downloading: Skript downloaden <gold>%s<reset>...
```

Comment:
what's the %s for here? not sure if the current sentence makes sense

Reply:
No idea for this one, looking at other .lang files and their relative sentence structures, I believe it to be correct